app_base.htm

```diff
@@ -0,0 +1,46 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="navbar_items">
+    <li class = ${'active' if request.dmp.page == 'index' else ''}><a href="/homepage/index/">Home</a></li>
```

Comment:
You should think about iterating over one li tag and just putting page information in a list.

Reply:
Oh that is a good idea!